packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -398,16 +402,23 @@ export const startCoinjoinSession =
     };
 
 // called from coinjoin account UI or exceptions like device disconnection, forget wallet/account etc.
-export const pauseCoinjoinSession = (account: Account) => (dispatch: Dispatch) => {
-    // get @trezor/coinjoin client if available
-    const client = getCoinjoinClient(account.symbol);
+export const pauseCoinjoinSession =
+    (accountKey: string) => (dispatch: Dispatch, getState: GetState) => {
+        const account = selectAccountByKey(getState(), accountKey);
 
-    // unregister account in @trezor/coinjoin
-    client?.unregisterAccount(account.key);
+        if (!account) {
+            return;
+        }
 
-    // dispatch data to reducer
-    dispatch(coinjoinSessionPause(account.key));
-};
+        // get @trezor/coinjoin client if available
+        const client = getCoinjoinClient(account?.symbol);
```

Comment:
nit: same here, redundant `?`
```suggestion
        const client = getCoinjoinClient(account.symbol);
```

Reply:
76c7b1d6d177798b40a8c34a8fb43461eb3aec34